scripts/suitec-analytics/lib/unloadTablesToS3.js

```diff
@@ -45,8 +46,9 @@ var prepareUnloadData = module.exports.prepareUnloadData = function(schemaName,
       return callback(err);
     }
 
+    var date = moment().format('MM-DD-YYYY');
```

Comment:
@sandeepmjay Should we try to start the switch to `YYYY-MM-DD` format here, or is it important that it stay the same as date formats elsewhere?